style.css

```diff
@@ -587,18 +595,27 @@ div {
   background-size: 130px;
 }
 
-#mobile-menu {
-  height: 85vh;
-  border-bottom-right-radius: 2em;
-  border-bottom-left-radius: 2em;
-  margin-bottom: 2em;
-  mix-blend-mode: normal;
-  box-shadow: 0 1em 1em rgba(37, 47, 137, 0.08);
+.header-nav-mobile {
+  position: absolute;
+  top: 0%;
+  right: 0%;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+  background-color: white;
+  color: #3a4a42;
+  border: 1px lightgrey solid;
+  border-radius: 5px;
 }
 
-#mobile-menu > ul {
-  clear: right;
-  padding-left: 0;
+.cancel-cross {
+  display: inline-block;
+  width: 24px;
+  height: auto;
+  position: absolute;
+  top: 10px;
+  left: 24%;
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_546](https://user-images.githubusercontent.com/69638013/203881242-30047360-3ea5-4a6d-9e2f-12dc8c55e1b8.png) | ![Screenshot_547](https://user-images.githubusercontent.com/69638013/203881300-1f1492af-d289-4c47-8742-03c2e6a8e46e.png)


- [ ] It would be better to  align the close button to look like the Figma design 👍🏿